### PR TITLE
Bintray

### PR DESCRIPTION
**Pull request by MasseGuillaume**
I published 0.1.0 on Bintray.

I requested to be added to https://bintray.com/sbt/sbt-plugin-releases so we won't need a ugly resolver.

@densh you will have to create a Bintray user so I can add you to the scala-native org that I created

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ In `project/plugins.sbt`:
 addSbtPlugin("org.scala-js" % "sbt-scalajs" % "0.6.13")
 
 resolvers += Resolver.sonatypeRepo("snapshots")                           // (1)
-addSbtPlugin("org.scala-native" % "sbt-cross"         % "0.1.0-SNAPSHOT") // (2)
+addSbtPlugin("org.scala-native" % "sbt-crossproject"         % "0.1.0-SNAPSHOT") // (2)
```
Comment (densh) [inline README.md]: Too much whitespace it seems.

File: README.md [hunk 2]
```diff
@@ -85,23 +85,23 @@ lazy val barNative = bar.native
...
 
 In `project/plugins.sbt`:
 
 ```scala
 addSbtPlugin("org.scala-js" % "sbt-scalajs" % "0.6.13")
 
 resolvers += Resolver.sonatypeRepo("snapshots")                           // (1)
-addSbtPlugin("org.scala-native" % "sbt-cross"         % "0.1.0-SNAPSHOT") // (2)
+addSbtPlugin("org.scala-native" % "sbt-crossproject"         % "0.1.0-SNAPSHOT") // (2)
 addSbtPlugin("org.scala-native" % "sbt-scalajs-cross" % "0.1.0-SNAPSHOT") // (3)
```
Comment (densh) [inline README.md]: `sbt-scalajs-crossproject`

File: README.md [hunk 3]
```diff
@@ -14,7 +14,7 @@ In `project/plugins.sbt`:
 addSbtPlugin("org.scala-js" % "sbt-scalajs" % "0.6.13")
 
 resolvers += Resolver.sonatypeRepo("snapshots")                           // (1)
```
Comment (densh) [inline README.md]: We don't need this any more.

**Comment by MasseGuillaume** [comment]
@densh resolved

**Comment by MasseGuillaume** [comment]
@densh can you re-enable travis? Ty ;-).

you can close this: #20

**Comment by MasseGuillaume** [comment]
CI is weird openning a new pr

**Pull request closed without merging**